docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -1,20 +1,29 @@
 # Prebuilt workspaces
 
-## Overview
+Prebuilt workspaces (Prebuilds) allow template administrators to improve the developer experience by reducing workspace
```

Comment:
I'm trying to stay away from the term `prebuilds` because it has confusing connotations (i.e. it's not a _build_, it's a _workspace_)
```suggestion
Prebuilt workspaces allow template administrators to improve the developer experience by reducing workspace
```

Reply:
TBH this is much wordier and less clear than "Prebuilt workspaces let you pre-provision and maintain a pool of ready-to-use workspaces." although I like the developer experience bit.

Reply:
yeah, I'm not super sold on `Prebuilds` either, I thought we were using it as a feature name, but I'm happy to just keep `prebuilt workspaces` throughout